scorer_cleanup

```diff
@@ -0,0 +1,33 @@
+#Declare custom scorer
+
+def RMSE(y_true,y_pred, verbose=False):
+    rmse = sqrt(mean_squared_error(y_true, y_pred))
+    if verbose==True:
+        print('RMSE: ', rmse)
+        return rmse
+    else:
+        return rmse
+
+def R2(y_true,y_pred, verbose=False):    
+    r2 = r2_score(y_true, y_pred)
+    if verbose==True:
+        print('R2: ', r2)
+        return r2
+    else:
+        return r2
+
+def two_score(y_true,y_pred, verbose=False):    
+    if verbose==True:
```

Comment:
the if block and the else block are actually the same if you simply pass verbose as an argument - e.g. `R2(y_true,y_pred, verbose)`

in case there's some misunderstanding about python functions, here is a simple (rough) description: each argument of the function can be used a variable within the function whether that argument is a list, dict, integer, or another function. by setting verbose=False in the definition of the function simply set that as the default value when nothing is passed to it. i.e. if you run `two_score(y, y_prime, True)`, then `verbose` would be `True` within that execution of the function